src/components/tx/TxData.js

```diff
@@ -280,6 +281,22 @@ class TxData extends React.Component {
     return tokenConfig.symbol;
   };
 
+  /**
+   * Get name of token from UID iterating through possible tokens in the transaction
+   *
+   * @param {string} uid UID of token to get the symbol
```

Comment:
chore: Minor correction to docs
```suggestion
   * @param {string} uid UID of token to get the name
```